src/styles/blocks/card.scss

```diff
@@ -0,0 +1,68 @@
+.card {
+  width: 200px;
+  padding: 0 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $bakcground-color;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
```

Comment:
according to checklist rule #8, you shouldn't hardcode the height of main container

Reply:
its not the main container? its img I suppose